css/main.css

```diff
@@ -1,3 +1,225 @@
 /* Add your CSS here */
+*{
 
+}
+
+body{
+    /* display: grid; */
+    /* grid-template-columns: ; */
+    /* min-width: auto; */
+    margin-top: 5rem;
+    font-family: "poppins", sans serif;
+  }
+
+.topnav{
+    display: flex;
+    flex-direction: row;
+     
+}
+
+#myLinks a {
```

Comment:
In general, try to avoid the use of ids in your CSS. This isn't an absolute rule, but a helpful hint for managing specificity when your designs become more complex. Read around this idea: https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4